Tests/bootstrap.php

```diff
@@ -1,4 +1,9 @@
 <?php
+// PHPUnit 6 introduced a breaking change that
+// removed PHPUnit_Framework_TestCase as a base class,
+// and replaced it with PHPUnit\Framework\TestCase
+if (!class_exists('\PHPUnit_Framework_TestCase') && class_exists('PHPUnit\Framework\TestCase'))
+    class_alias('PHPUnit\Framework\TestCase', '\PHPUnit_Framework_TestCase');
```

Comment:
Please add curly braces around the statement. See [PSR-2](http://www.php-fig.org/psr/psr-2/#control-structures).